ui/src/components/mobileservices/BindingPanel.js

```diff
@@ -65,9 +71,12 @@ class BindingPanel extends Component {
     
           return <Form schema={this.state.schema}
```

Comment:
I should have used this in Create Build Config form instead of implementing it by myself, it could saved me much time.